run-tests.php

```diff
@@ -1432,7 +1432,7 @@ function run_all_tests_parallel(array $test_files, array $env, $redir_tested): v
     $startTime = microtime(true);
     for ($i = 1; $i <= $workers; $i++) {
         $proc = proc_open(
-            [$thisPHP, $thisScript],
+            [$thisPHP, "-dopcache.jit=0", $thisScript],
```

Comment:
why? too slow?

Reply:
During development I found that ``run-tests.php`` workers run with JIT enabled even if main ``run-tests.php -jN`` is always launched with JIT disabled. This caused problems at time when new JIT wasn't good enough.

Reply:
oh that makes sense. well if it's good enough now, i think it makes most sense to leave it at default, whatever php.ini says it should be?

Reply:
o i c. how about
```php
            [$thisPHP, "-dopcache.jit=".((int)ini_get("dopcache.jit")), $thisScript],
```

Reply:
@arnaud-lb, @iluuu1994  what do you think?

Reply:
I would usually do `php run-tests.php -p sapi/cli/php` if there's some bug in the binary that prevents run-tests.php from being executed. If this works with the new JIT now, I'd prefer to revert this line.

Reply:
Reverted.